test/Extensions/ServiceBus.Tests/Streaming/EHStreamProviderCheckpointTests.cs

```diff
@@ -84,14 +84,14 @@ public void Configure(IConfiguration configuration, IClientBuilder clientBuilder
             }
         }
 
-        [SkippableFact]
+        [SkippableFact(Skip="Investigating intermitent failures")]
```

Comment:
I think a better pattern is to open an issue with names of the intermittently failing tests listed and put the URL to that issue as the description for `Skip`. For example, like in https://github.com/dotnet/orleans/blob/master/test/TesterInternal/GeoClusterTests/MultiClusterNetworkTests.cs#L149

Reply:
will do